2024-12-13 05:32:33,869 - INFO - Authenticating with Hugging Face...
2024-12-13 05:32:34,130 - ERROR - Authentication failed: Invalid user token. If you didn't pass a user token, make sure you are properly logged in by executing `huggingface-cli login`, and if you did pass a user token, double-check it's correct.. Exiting...
2024-12-13 05:32:59,870 - INFO - Authenticating with Hugging Face...
2024-12-13 05:33:00,055 - INFO - Authenticated with Hugging Face successfully.
2024-12-13 05:33:00,056 - INFO - Loading 'anudesh' config with 'hi' split from Hugging Face...
2024-12-13 05:33:03,198 - INFO - Successfully loaded 'anudesh' config with 'hi' split containing 7577 rows.
2024-12-13 05:33:03,754 - INFO - Successfully translated row 1/7577.
2024-12-13 05:33:03,759 - INFO - Progress saved to translated_dataset.csv.
2024-12-13 05:33:05,145 - ERROR - Git error during push: Command '['git', 'push']' returned non-zero exit status 128.
2024-12-13 05:33:05,145 - WARNING - Retrying Git operations in the next iteration...
2024-12-13 05:33:05,646 - INFO - Successfully translated row 2/7577.
2024-12-13 05:33:05,651 - INFO - Progress saved to translated_dataset.csv.
2024-12-13 05:33:07,224 - ERROR - Git error during push: Command '['git', 'push']' returned non-zero exit status 128.
2024-12-13 05:33:07,224 - WARNING - Retrying Git operations in the next iteration...
2024-12-13 05:33:07,722 - INFO - Successfully translated row 3/7577.
2024-12-13 05:33:07,727 - INFO - Progress saved to translated_dataset.csv.
2024-12-13 05:33:09,063 - ERROR - Git error during push: Command '['git', 'push']' returned non-zero exit status 128.
2024-12-13 05:33:09,063 - WARNING - Retrying Git operations in the next iteration...
2024-12-13 05:33:09,578 - INFO - Successfully translated row 4/7577.
2024-12-13 05:33:09,582 - INFO - Progress saved to translated_dataset.csv.
2024-12-13 05:33:11,128 - ERROR - Git error during push: Command '['git', 'push']' returned non-zero exit status 128.
2024-12-13 05:33:11,128 - WARNING - Retrying Git operations in the next iteration...
2024-12-13 05:33:11,603 - INFO - Successfully translated row 5/7577.
2024-12-13 05:33:11,607 - INFO - Progress saved to translated_dataset.csv.
2024-12-13 05:33:13,117 - ERROR - Git error during push: Command '['git', 'push']' returned non-zero exit status 128.
2024-12-13 05:33:13,118 - WARNING - Retrying Git operations in the next iteration...
2024-12-13 05:33:13,604 - INFO - Successfully translated row 6/7577.
2024-12-13 05:33:13,609 - INFO - Progress saved to translated_dataset.csv.
